vochain/start.go

```diff
@@ -309,6 +304,23 @@ func newTendermint(app *BaseApplication,
 		}
 	}
 
+	// We need to fetch chain_id in order to make Replay work,
+	// since signatures depend on it.
+	log.Warnf("genesis file at %s", tconfig.GenesisFile())
```

Comment:
shouldn't this be Info or Debug? (i imagine this being a leftover from debugging)

Reply:
right